src/components/data-exploration/CellInfo.jsx

```diff
@@ -35,23 +36,26 @@ const CellInfo = (props) => {
       }}
     >
       {cellInfo.cellName ? (
-        <div>
-          Cell id:&nbsp;
-          {cellInfo.cellName}
+        <div style={{ fontSize: '0.75rem' }}>
+          {`Cell id: ${cellInfo.cellName}`}
         </div>
       ) : <></>}
       {cellInfo.geneName ? (
-        <div>
-          Gene name:&nbsp;
-          {cellInfo.geneName}
+        <div style={{ fontSize: '0.75rem' }}>
+          {`Gene name: ${cellInfo.geneName}`}
         </div>
       ) : <></>}
       {cellInfo.expression !== undefined ? (
-        <div>
+        <div style={{ fontSize: '0.75rem' }}>
           Expression Level:&nbsp;
           {parseFloat(cellInfo.expression.toFixed(3))}
         </div>
       ) : <></>}
+      {cellInfo.cellSets?.length > 0 ? cellInfo.cellSets.map((cellSetName) => (
+        <div style={{ fontSize: '0.75rem' }}>
+          {`${_.truncate(cellSetName)}`}
```

Comment:
do you need the `${}` here, why not just {_.truncate(cellSetName)}?

Reply:
oh yea, I don't need that. Removed.